### DISCUSSION TITLE
Trigger progressbar manually?

### DISCUSSION THREAD

**Discussion by Crinchy** [NdungutseCharles103/next13-progressbar#26] [General]
Hey, is it possible to trigger the progress bar manully on button submits instead of only page switches?

**Comment by ndungtse** [comment]
That can be possible, but when will it actually stop? will someone have to stop it again manually?

**Reply by Crinchy** [reply]
Yes, i guess, i would start and stop it manually then. How can i implement it, whats the function to trigger it?

**Reply by ndungtse** [reply]
Currently there is no function available for it but it can be added in future todos

**Reply by Crinchy** [reply]
It could also be cool, if you could enable that the progress bar is also showing on navigation back, while using the router.back() function in next.js, i would appreciate it alot.
